src/homepage/howTo/custom-listeners.html

```diff
@@ -121,65 +112,14 @@ <h3><b>Starting James with BigMessageListener</b></h3>
 &lt;/listeners&gt;
                 </code></pre>
 
-                <p>Finally, starting a James Server by docker compose</p>
-                <p>Getting James docker-compose.yml</p>
-                <pre><code>
-$ wget https://raw.githubusercontent.com/apache/james-project/master/server/apps/jpa-app/docker-compose.yml
-                </code></pre>
-                <p>Using James Guice JPA instead of the default product</p>
-                <pre><code>
-services:
-  james:
-    ...
-    image: apache/james:jpa-latest
-                </code></pre>
-                <p>Add the following volumes for james service:</p>
-                <pre><code>
-volumes:
-  - $PWD/conf:/root/conf/
-                </code></pre>
+                <p>Finally, we will start a James server using that. We will rely on docker default image for simplicity.
+                    We need to be using the <code>listeners.xml</code> configuration that we had been writing and position the jar in the extensions-jars folder (specific to guice).
+                    This can be achieved with the following command:</p>
 
-                <p>
-                    When you are using your listeners, place the jar containing your listeners into "extensions-jars" directory
-                    and add a volume for it.
-                </p>
-                <pre><code>
-volumes:
-  - $PWD/extensions-jars:/root/extensions-jars/
-                </code></pre>
-                <p>
-                    Putting compiled jars into extensions-jars directory
```

Comment:
Why removing the section explaining how to build and put custom jar into James?

Reply:
I added it above 
```
We need to be using the <code>listeners.xml</code> configuration that we had been writing and position the jar in the extensions-jars folder (specific to guice).```
```
And added the docker command (replace for docker-compose example)
```
docker run -p "25:25" -p "143:143" \
               -v "$PWD/src/main/resources/listeners.xml:/root/conf/listeners.xml" \
               -v "$PWD/target/custom-listeners:/root/extensions-jars/custom-listeners" \
        apache/james:memory-latest --generate-keystore
```

Make it simpler, and similar to other howTo documents (e.g.: custom-mailets, custom-healthchecks...)